tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
But that's just the playback mode, not the performance mode. The only metric we have is total time, which means we have to have this code or else we're only measuring the time it takes to move buffers around.